weaviate/misc/misc.py

```diff
@@ -0,0 +1,37 @@
+from weaviate.connect import Connection
+from weaviate.exceptions import (
+    UnexpectedStatusCodeException,
+    RequestsConnectionError,
+    EmptyResponseException
+)
+
+class Misc:
```

Comment:
Can be have a better name for this? :)

Reply:
Yes, however this is in parity with the other 3 clients. If we make this change, we should apply it to all clients